datajunction-query/djqs/engine.py

```diff
@@ -65,6 +65,22 @@ def get_columns_from_description(
 
     return columns
 
+def create_trino_engine(engine: Engine) -> 'Engine':
+    """
+    Create a SQLAlchemy engine for Trino.
+    """
+    uri = engine.uri
+    user = engine.extra_params['user']
+    password = engine.extra_params['password']
+    scheme, rest = uri.split("://")
+    host_port, catalog_schema = rest.split("/", 1)
+    catalog , schema = catalog_schema.split('/')
+    host, port = host_port.split(":")
+    if scheme == "https":
+        engine_str = f"trino://{user}:{password}@{host}:{port}/{catalog}/{schema}?protocol={scheme}"
+        return create_engine(engine_str)
+    else:
+        return create_engine(engine.uri,engine.extra_params)
```

Comment:
There can probably be a check here to make sure that the engine.uri actually starts with `trino` right?